### PR TITLE
feat: connect submission form to backend

### PR DESCRIPTION
**Pull request by jc-tung**
enhances #22

### CODE REVIEW

File: frontend/src/components/LandingPage/LandingPage.tsx [hunk 1]
```diff
@@ -70,6 +71,11 @@ interface Verification {
 const LandingPage: React.FC<Verification> = ({verify, isVerified}: Verification) => {
   const [showAlert, setShowAlert] = useState(false);
   const [alertText, setAlertText] = useState("");
+  // const [userId, setUserId] = useState(-1);
+
+  // useEffect(() => {
+  //   localStorage.setItem("userId", userId.toString());
+  // }, [userId]);
```
Comment (nareha) [inline frontend/src/components/LandingPage/LandingPage.tsx:78]: Nit: Remove unused code comments.

File: backend/db/db.ts [hunk 2]
```diff
@@ -91,21 +91,44 @@ export async function addUser(email: string): Promise<string> {
...
+        submission.interval_end, submission.source, submission.destination,
+        submission.contact, submission.max_group_size);
+      insertStmt.finalize();
+      console.log('Inserted new row for user ', submission.userid);
+    }
+  });
+  return new Promise<number>((resolve, reject) => {resolve(1)});
 }
 
 // TODO(joycetung): implement filtering support.
```
Comment (nareha) [inline backend/db/db.ts:134]: Nit: Remove unused code comments.

File: frontend/src/structures.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  
+export type Submission = {
+    userid: number;
+    interval_start: Date;
+    interval_end: Date;
+    source: Location;
+    destination: Location;
+    contact: string;
+    max_group_size?: number;
+};
```
Comment (nareha) [inline frontend/src/structures.ts:15]: Nit: Add new line at EOF.

Comment (jc-tung) [reply frontend/src/structures.ts]: done.

**Review by alanhwu — APPROVED** [review]

**Pull request merged**